server/service/src/backend_plugin/boajs/methods/log.rs

```diff
@@ -0,0 +1,16 @@
+use boa_engine::*;
+use log::info;
+
+// Log method will log all arguments
+pub(crate) fn bind_method(context: &mut Context) -> Result<(), JsError> {
+    context.register_global_callable(
+        JsString::from("log"),
+        0,
+        NativeFunction::from_copy_closure(move |_, args, _| {
+            info!("from js {:#?}", args);
```

Comment:
Would be nice to handle strings here as a special case to make the log a bit cleaner.

```
ebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36" 0.039822
2025-01-06 17:16:32.156279000 [INFO ] <service::backend_plugin::boajs::methods::log:10>:from js [
    String(
        "Running SQL query",
    ),
]
```

Reply:
> Would be nice to handle strings here as a special case to make the log a bit cleaner.

Logging is only for devs, yes it would be great to handle a bit better here, but it's could be any array of args, and extracting string and formatting them differently seems like quite a bit of code. I feel like the display is adequate for the target consumer. 

Please let me know if you feeling strong on this one